src/mount/writedata.cc

```diff
@@ -367,7 +367,7 @@ void write_job_delayed_end(inodedata* id, int status, int seconds, Glock &lock)
 		write_cb_release_blocks(id->dataChain.size(), lock);
 		id->dataChain.clear();
 		id->inqueue = false;
-		id->maxfleng = 0; // proper file length is now on the master server, remove our length cache
+		// We don't reset maxfleng for a while longer
```

Comment:
Is this comment correct here or should it be placed below inside the if statement?

Reply:
Done.